share/governance.md

```diff
@@ -200,10 +234,12 @@ to its success. To that end, decision making must be done in a transparent,
 open fashion. No decisions about the project’s direction, bug fixes or features
 may be done without community involvement and participation. Discussions must
 begin at the earliest possible point on a topic; the community’s participation
-is vital during the entire decision-making process.
+is vital during the entire decision-making process. While conversations happen
+on many mediums, decisions must be made through the mailing list.
```

Comment:
Make it explicit that the mailing list is where things are decided.

Reply:
in the past we used the mailing list or github issues/pr. I think we used github more than the mailinglist, but that's really just a feeling and I don't have a strong preference here. I think it's slightly easier to count votes on github because there's a thums up/down option.

Reply:
Right, we might use the mailing list to drive people to github or https://civs1.civs.us/ or wherever to get inputs, though the decisions should be conveyed in one place so that people understand what is happening without having to be on all the mediums.

Reply:
Yes, That's also how I used it in the past. Created a GitHub PR and requested feedback on the mailinglist and on IRC.

Reply:
I feel like GitHub is the place where we should ask people to record their thoughts (as we get permanent records of that data) and then we have communications to Slack/IRC and the ML. 

I personally hate mailing lists and don't really use them, so anything going direct to ML has a high chance of getting missed for me.

Reply:
Mailing lists can allow for a low impact approach such as an -announce style list. This means you can stay up to date without having to know that conversations are happening in multiple github PR's, issues, discussions, etc.